### PR TITLE
[ROCKETMQ-104] Make MQAdmin commands throw exceptions

### PR DESCRIPTION
**Pull request by shroman**
JIRA issue: https://issues.apache.org/jira/browse/ROCKETMQ-104

### CODE REVIEW

**Comment by zhouxinyu** [comment]
Hi @shroman ,
LGTM, but why do we add so many `@Ignore` annotations in this PR? All the he unit tests of tools module have been skipped in `tools/pom.xml`. We may forget to remove these annotations after this issue[1] has been resolved.

[1]. https://issues.apache.org/jira/browse/ROCKETMQ-77

**Comment by shroman** [comment]
@zhouxinyu Thanks for checking it!
Well, `@Ignore` is there just for finer granularity -- we remove it one by one when implementing each test. If you think we don't need it, I can remove ;)

**Comment by zhouxinyu** [comment]
please @vongosling @lizhanhui help review.

File: tools/src/main/java/org/apache/rocketmq/tools/command/SubCommand.java [hunk 1]
```diff
@@ -21,11 +21,11 @@
 import org.apache.rocketmq.remoting.RPCHook;
 
 public interface SubCommand {
-    public String commandName();
+    String commandName();
```
Comment (vongosling) [inline tools/src/main/java/org/apache/rocketmq/tools/command/SubCommand.java]: Cool. JLS points out that the interface method is public abstract

File: tools/src/main/java/org/apache/rocketmq/tools/command/offset/ResetOffsetByTimeCommand.java [hunk 2]
```diff
@@ -22,25 +22,16 @@
...
 import org.apache.rocketmq.common.message.MessageQueue;
 import org.apache.rocketmq.common.protocol.ResponseCode;
 import org.apache.rocketmq.remoting.RPCHook;
-import org.apache.rocketmq.srvutil.ServerUtil;
 import org.apache.rocketmq.tools.admin.DefaultMQAdminExt;
 import org.apache.rocketmq.tools.command.SubCommand;
+import org.apache.rocketmq.tools.command.SubCommandException;
 
 public class ResetOffsetByTimeCommand implements SubCommand {
-    public static void main(String[] args) {
```
Comment (vongosling) [inline tools/src/main/java/org/apache/rocketmq/tools/command/offset/ResetOffsetByTimeCommand.java]: +1

**Comment by vongosling** [comment]
@shroman IMO, we can let the tool module unit-test passed once someone polish here. thoughts?

**Comment by shroman** [comment]
@vongosling Thanks for reviewing!
Sure, unit tests can be completed after we improve this module.

**Comment by zhouxinyu** [comment]
Hi, @shroman  this PR will be merged to `develop` branch firstly as our new branching model: http://rocketmq.incubator.apache.org/docs/branching-model

**Comment by vongosling** [comment]
@shroman Could you help to close this PR and Jira issue:-)

**Pull request closed without merging**